rdr_service/dao/metrics_cache_dao.py

```diff
@@ -1517,6 +1527,7 @@ def get_metrics_cache_sql(self, hpo_id):
                                 FROM {temp_table_name} ps
                                 LEFT JOIN participant_race_answers q ON ps.participant_id = q.participant_id
                                 WHERE ps.questionnaire_on_the_basics = 1
+                                    OR (ps.questionnaire_on_the_basics = 0 AND q.code_id is NULL)
```

Comment:
if `ps.questionnaire_on_the_basics = 0`, q.code_id will always be NULL.
I think it's the same result if we just remove this where clause.

Reply:
Done